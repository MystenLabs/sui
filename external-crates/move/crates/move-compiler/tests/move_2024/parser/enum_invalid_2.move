module 0x42::m {
    public enum Expression {
       Done(x: u64),
       Add { u64 },
    }
}
