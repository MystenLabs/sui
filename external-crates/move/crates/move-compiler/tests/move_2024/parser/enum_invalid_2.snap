error[E01002]: unexpected token
  ┌─ tests/move_2024/parser/enum_invalid_2.move:3:13
  │
3 │        Done(x: u64),
  │             ^ Cannot use named fields in a positional definition

error[E01002]: unexpected token
  ┌─ tests/move_2024/parser/enum_invalid_2.move:4:18
  │
4 │        Add { u64 },
  │                  ^
  │                  │
  │                  Unexpected '}'
  │                  Expected ':'

